web/src/Overview.jsx

```diff
@@ -31,12 +31,14 @@ import Category from "./Category";
 import LanguageSelector from "./LanguageSelector";
 import Storage from "./Storage";
 import Users from "./Users";
+import Network from "./Network";
 import Popup from "./Popup";
 
 import {
   EOS_SOFTWARE as OverviewIcon,
   EOS_TRANSLATE as LanguagesSelectionIcon,
   EOS_VOLUME as HardDriveIcon,
+  EOS_SETTINGS_ETHERNET as NetworkIcon,
```

Comment:
A better icon is more than welcome. But remember that we are using https://eos-icons.com/, so have a look there for consistence. (Spoiler: although it is listed there, EOS_LAN actually is not available; in fact, would be nice to use it to visually identify listed wired connections)